CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.5.0 / 2016-11-01
+
+This release requires a storage wipe. It contains fundamental internal
+changes that came with implementing the high availability mode.
+
+* [FEATURE] Alertmanager clustering for high availability
+* [FEATURE] Garbage collection of old silences and notification logs
+* [CHANGE] New storage format
+* [CHANGE] Stricter silence semantics for consistent historical view
+
 ## v0.5.0-beta.0 / 2016-10-06
```

Comment:
We had pre releases out for quite some time for the Prometheus server and still had one consolidated block of changes for the final release.

You could point out in the announcement mail what changed compared to the last beta version.